pkg/api/v1/types.go

```diff
@@ -589,6 +594,12 @@ type CephFSVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// FlockerVolumeSource represents a Flocker volume attached to a kubelet's host machine and exposed to the pod for its usage.
```

Comment:
Same nit.  How about:

`FlockerVolumeSource represents a Flocker volume mounted by the Flocker agent`?